### PR TITLE
March Ratification Poll Outcomes

### PR DESCRIPTION
**Pull request by blimpa**
@patrick-j-govalpha , please review :)

### CODE REVIEW

File: MIP101/MIP101.md [hunk 1]
```diff
@@ -11,13 +11,13 @@ Author(s): @rune
 Contributors:
 Tags: constitution, endgame, pregame, constitution-mip-set
 Type: General
-Status: Formal Submission
+Status: Accepted
 Date Proposed: 2023-01-31
-Date Ratified: <yyyy-mm-dd>
+Date Ratified: 2023-03-07
```
Comment (Gala-G) [inline MIP101/MIP101.md]: 2023-03-27

File: MIP102/MIP102.md [hunk 2]
```diff
@@ -9,11 +9,12 @@ Author(s): @rune
 Contributors:
 Tags: endgame, pregame, constitution-mip-set
 Type: General
-Status: Formal Submission
+Status: Accepted
 Date Proposed: 2023-02-06
-Date Ratified: <yyyy-mm-dd>
-Dependencies: 
+Date Ratified: 2023-03-07
```
Comment (Gala-G) [inline MIP102/MIP102.md]: same
2023-03-27

File: MIP102/MIP102c2-Subproposals/MIP102c2-SP1.md [hunk 3]
```diff
@@ -8,11 +8,11 @@ MIP(s) to be amended: MIP0, MIP16, MIP24, MIP39, MIP40, MIP41, MIP51, MIP55
 Author(s): Rune
 Contributors:
 Tags: constitution, endgame, constitution-mip-set
-Status: Formal Submission
-Date Proposed: 2023-02-6
-Date Ratified: N/A
+Status: Accepted
+Date Proposed: 2023-02-06
+Date Ratified: 2023-03-07
```
Comment (Gala-G) [inline MIP102/MIP102c2-Subproposals/MIP102c2-SP1.md]: same
2023-03-27

File: MIP103/MIP103.md [hunk 4]
```diff
@@ -8,13 +8,13 @@ Author(s): @rune
 Contributors:
 Tags: endgame, scope-framework, constitution-mip-set
 Type: General
-Status: Formal Submission
+Status: Accepted
 Date Proposed: 2023-02-06
-Date Ratified: <yyyy-mm-dd>
+Date Ratified: 2023-03-07
```
Comment (Gala-G) [inline MIP103/MIP103.md]: same
2023-03-27

File: MIP14/MIP14c2-Subproposals/MIP14c2-SP4.md [hunk 5]
```diff
@@ -6,11 +6,12 @@ MIP14c2-SP#: 4
...
 Tags: Starknet
-Status: Formal Submission
+Status: Obsolete
 Date Proposed: 2023-01-12
-Date ratified: 
+Date ratified: N/A
 Forum URL: https://forum.makerdao.com/t/mip14c2-sp4-starknet-cu-budget-extension/19403/1
-Ratification Poll URL: TBD
+Ratification Poll URL: https://vote.makerdao.com/polling/QmXUiHqH
+Extra: This proposal has been marked as obsolete as per the changes introduced in [MIP0](https://mips.makerdao.com/mips/details/MIP0) via [MIP102c2-SP1](https://mips.makerdao.com/mips/details/MIP102c2SP1).
```
Comment (Gala-G) [inline MIP14/MIP14c2-Subproposals/MIP14c2-SP4.md:14]: For consistency, write: Extra: This MIP has been made obsolete by the passage of [MIP102c2-SP1](https://mips.makerdao.com/mips/details/MIP102c2SP1)

This was what we wrote in all the MIPs we rendered obsolete.

File: MIP14/MIP14c2-Subproposals/MIP14c2-SP5.md [hunk 6]
```diff
@@ -9,9 +9,10 @@ Contributors:
 Tags: tech-001, core-unit,
 Status: Formal Submission
 Date Proposed: 2023-02-01
-Date ratified: -
+Date ratified: N/A
 Forum URL: https://forum.makerdao.com/t/mip14c2-sp5-requesting-protocol-dai-transfer-to-fund-techops-cu-for-march-2023/19624
-Ratification Poll URL: 
+Ratification Poll URL: https://vote.makerdao.com/polling/Qmdzxxw4
+Extra: This proposal has been marked as obsolete as per the changes introduced in [MIP0](https://mips.makerdao.com/mips/details/MIP0) via [MIP102c2-SP1](https://mips.makerdao.com/mips/details/MIP102c2SP1).
```
Comment (Gala-G) [inline MIP14/MIP14c2-Subproposals/MIP14c2-SP5.md:15]: For consistency, write: Extra: This MIP has been made obsolete by the passage of [MIP102c2-SP1](https://mips.makerdao.com/mips/details/MIP102c2SP1)

This was what we wrote in all the MIPs we rendered obsolete.

File: MIP40/MIP40c3-Subproposals/MIP40c3-SP88.md [hunk 7]
```diff
@@ -7,13 +7,14 @@ MIP40c3-SP: #88
...
+Status: Obsolete
 Date Applied: 2022-12-02
-Date Ratified: <yyyy-mm-dd>
+Date Ratified: N/A
 Budget Start Date: 2023-04-01
 Budget End Date: 2024-03-31
 Forum URL: https://forum.makerdao.com/t/mip40c3-spxx-techops-core-unit-dai-budget/19017
-Ratification Poll URL:
+Ratification Poll URL: https://vote.makerdao.com/polling/QmWpo8BJ
+Extra: This budget has been marked as obsolete as per the changes introduced in [MIP40](https://mips.makerdao.com/mips/details/MIP40) via [MIP102c2-SP1](https://mips.makerdao.com/mips/details/MIP102c2SP1).
```
Comment (Gala-G) [inline MIP40/MIP40c3-Subproposals/MIP40c3-SP88.md:17]: For consistency, write: Extra: This MIP has been made obsolete by the passage of [MIP102c2-SP1](https://mips.makerdao.com/mips/details/MIP102c2SP1)

This was what we wrote in all the MIPs we rendered obsolete.

File: MIP40/MIP40c3-Subproposals/MIP40c3-SP90.md [hunk 8]
```diff
@@ -7,13 +7,14 @@ MIP40c3-SP#: 90
...
+Status: Obsolete
 Date Applied: 2023-02-07
-Date Ratified: <yyyy-mm-dd>
+Date Ratified: N/A
 Budget Start Date: 2023-04-01
 Budget End Date: 2024-03-31
 Forum URL: https://forum.makerdao.com/t/mip40c3-sp90-govalpha-core-unit-dai-budget-2023-2024/19712
-Ratification Poll URL:
+Ratification Poll URL: https://vote.makerdao.com/polling/QmXoH4rE
+Extra: This budget has been marked as obsolete as per the changes introduced in [MIP40](https://mips.makerdao.com/mips/details/MIP40) via [MIP102c2-SP1](https://mips.makerdao.com/mips/details/MIP102c2SP1).
```
Comment (Gala-G) [inline MIP40/MIP40c3-Subproposals/MIP40c3-SP90.md:17]: For consistency, write: Extra: This MIP has been made obsolete by the passage of [MIP102c2-SP1](https://mips.makerdao.com/mips/details/MIP102c2SP1)

This was what we wrote in all the MIPs we rendered obsolete.

File: MIP40/MIP40c3-Subproposals/MIP40c3-SP91.md [hunk 9]
```diff
@@ -7,11 +7,12 @@ MIP40c3-SP#: 91
...
 Tags: core-unit, cu-gov-001, budget, mkr-budget
-Status: Formal Submission
+Status: Obsolete
 Date Proposed: 2023-02-07
-Date Ratified: yyyy-mm-dd
+Date Ratified: N/A
 Forum URL: https://forum.makerdao.com/t/mip40c3-sp91-govalpha-core-unit-mkr-budget-q2-2023/19713
-Ratification Poll URL:
+Ratification Poll URL: https://vote.makerdao.com/polling/QmWTwKiQ
+Extra: This budget has been marked as obsolete as per the changes introduced in [MIP40](https://mips.makerdao.com/mips/details/MIP40) via [MIP102c2-SP1](https://mips.makerdao.com/mips/details/MIP102c2SP1).
```
Comment (Gala-G) [inline MIP40/MIP40c3-Subproposals/MIP40c3-SP91.md:15]: idem

File: MIP40/MIP40c3-Subproposals/MIP40c3-SP92.md [hunk 10]
```diff
@@ -7,13 +7,14 @@ MIP40c3-SP#: 92
...
+Status: Obsolete
 Date Applied: 2023-02-08
-Date Ratified:
+Date Ratified: N/A
 Budget Start Date: 2023-03-01
 Budget End Date: 2024-02-29
 Forum URL: https://forum.makerdao.com/t/mip40c3-sp92-modify-risk-core-unit-budget-risk-001/19741
-Ratification Poll URL:
+Ratification Poll URL: https://vote.makerdao.com/polling/QmSsENPv
+Extra: This budget has been marked as obsolete as per the changes introduced in [MIP40](https://mips.makerdao.com/mips/details/MIP40) via [MIP102c2-SP1](https://mips.makerdao.com/mips/details/MIP102c2SP1).
```
Comment (Gala-G) [inline MIP40/MIP40c3-Subproposals/MIP40c3-SP92.md:17]: idem

File: MIP40/MIP40c3-Subproposals/MIP40c3-SP96.md [hunk 11]
```diff
@@ -5,13 +5,14 @@
...
+Status: Obsolete
 Date Applied: 2023-02-08
-Date Ratified: 
+Date Ratified: N/A
 Budget Start Date: 2023-05-01 
 Budget End Date: 2024-04-31 
 Forum URL: https://forum.makerdao.com/t/mip40c3-sp96-modify-data-insights-core-unit-budget-din-001/19748
-Ratification Poll URL:
+Ratification Poll URL: https://vote.makerdao.com/polling/QmUqdiXM
+Extra: This budget has been marked as obsolete as per the changes introduced in [MIP40](https://mips.makerdao.com/mips/details/MIP40) via [MIP102c2-SP1](https://mips.makerdao.com/mips/details/MIP102c2SP1).
```
Comment (Gala-G) [inline MIP40/MIP40c3-Subproposals/MIP40c3-SP96.md:15]: idem

File: MIP55/MIP55c3-Subproposals/MIP55c3-SP16.md [hunk 12]
```diff
@@ -7,12 +7,13 @@ MIP55c3-SP#: 16
...
 Tags:
-Status: Formal Submission
+Status: Obsolete
 Date Proposed: 2023-02-08
-Date Ratified: 
+Date Ratified: N/A
 Amount Requested: 347,100 DAI
 Recipient Address: 0xD9847E6b1314f0327F320E43B51ca0AaAD6FF509
 Forum URL: https://forum.makerdao.com/t/mip55c3-sp16-phoenix-labs-spark-lend-launch/19734
+Extra: This proposal has been marked as obsolete as per the changes introduced in [MIP55](https://mips.makerdao.com/mips/details/MIP55) via [MIP102c2-SP1](https://mips.makerdao.com/mips/details/MIP102c2SP1).
```
Comment (Gala-G) [inline MIP55/MIP55c3-Subproposals/MIP55c3-SP16.md:16]: For consistency, write: Extra: This MIP has been made obsolete by the passage of [MIP102c2-SP1](https://mips.makerdao.com/mips/details/MIP102c2SP1)

This was what we wrote in all the MIPs we rendered obsolete.

**Review by Gala-G — COMMENTED** [review]
@blimpa 
the ratification dates until MIP114 are wrong, it's 2023-03-27.
Also, the extra field explaining why the MIP was rendered obsolete, I think it's better to write down: "Extra: This MIP has been made obsolete by the passage of [MIP102c2-SP1](https://mips.makerdao.com/mips/details/MIP102c2SP1)"
... as we did for  all the MIPs we rendered obsolete.

thank you

@patrick-j-govalpha

File: MIP14/MIP14c2-Subproposals/MIP14c2-SP4.md (hunk 5 above)
Comment (ghost) [reply MIP14/MIP14c2-Subproposals/MIP14c2-SP4.md:14]: IMO there should be a difference between MIPs and subproposals, so I think we should keep this as is

**Review by ghost — APPROVED** [review]

File: MIP14/MIP14c2-Subproposals/MIP14c2-SP4.md (hunk 5 above)
Comment (Gala-G) [reply MIP14/MIP14c2-Subproposals/MIP14c2-SP4.md:14]: I see. Makes sense. Thank you!

**Pull request merged**